test/spmd/tensor/test_matrix_ops.py

```diff
@@ -1,31 +1,84 @@
 # Copyright (c) Meta Platforms, Inc. and affiliates
```

Comment:
Looks like the change comes with base change. Once the base got merged, I will rebase.